cherrypy/test/test_static.py

```diff
@@ -215,13 +215,13 @@ def test_config_errors(self):
         self.assertErrorPage(500)
         if sys.version_info >= (3, 3):
             errmsg = (
-                'TypeError: staticdir\(\) missing 2 '
+                r'TypeError: staticdir\(\) missing 2 '
```

Comment:
The string in `helper.py` touched by this change is not an actual docstring AFAIU. It's just a noop multiline string literal within the body of the `start()` method (which has another actual docstring not touched by this change). I believe another (better?) fix for the other thing would be to convert it to a comment rather than even trying to make it a valid string.